docs/_articles/Where_Is_My_Bundle.md

```diff
@@ -1,17 +1,14 @@
+---
+layout: default
+title: Where Is My Bundle
+summary: Short description of steps that you can take to debug problems when you don't see your bundle.
+---
+
 Where Is My Bundle
 ==================
 
 Sometimes when you start Eclipse, you don't see your bundle and you wonder why. This page will provide a short description of steps that you can take to debug this problem.
 
-Contents
```

Comment:
Can this somehow be retained or added to the navigation index?

Reply:
Added TOC (Table of contents) to articles in https://github.com/eclipse-equinox/equinox/pull/849/commits/3ad79338c3cabcec1433450f27fe86219330c47a

See https://kramdown.gettalong.org/converter/html.html#toc

Unfortunately we need to add it in each post / article at the moment. I tried doing it in the layout, but it didn't work.